src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -48,25 +48,41 @@ public void storeUser(User user) {
     Entity userEntity = new Entity("User", user.getEmail());
     userEntity.setProperty("email", user.getEmail());
     userEntity.setProperty("aboutMe", user.getAboutMe());
+    userEntity.setProperty("displayedName", user.getDisplayedName());
     datastore.put(userEntity);
   }
 
+  public void Test() {
+    Query query = new Query("User");
+    PreparedQuery results = datastore.prepare(query);
+
+    int cnt = 0;
+    for(Entity entity : results.asIterable()) {
+      System.out.println((String)entity.getProperty("email") + " - " + (String)entity.getProperty("aboutMe") + " - " + (String)entity.getProperty("displayedName"));
+      ++cnt;
+    }
+    System.out.println(cnt);
+  }
+
   /**
    * Returns the User owned by the email address, or
    * null if no matching User was found.
    */
   public User getUser(String email) {
+    Test();
```

Comment:
Should this be removed?

Reply:
Yes, I was in hurry to summit the pull request before the deadline and forget to remove it.